 *******> update.4

 Author: Stephan Schott-Verdugo

 Date: August 12, 2024

 Programs: packmol-memgen

 Description: Fix search of pdb2pqr in PATH

 --------------------------------------------------------------------------------
 AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py
index 893feedbdb..779a608e07 100755
--- AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py
+++ AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py
@@ -15,7 +15,7 @@ try:
 #    from pdb2pqr.main import main_driver as pdb2pqr
 #    from pdb2pqr.main import build_main_parser as pdb2pqr_args
     import shutil
-    pdb2pqr = shutil.which("pdb2pqr30")
+    pdb2pqr = shutil.which("pdb2pqr30")  or shutil.which("pdb2pqr")
 except:
     logger.debug("PDB2PQR not available. Protonation with --pdb2pqr will not be available")
     pdb2pqr = False
